.github/workflows/windows.yml

```diff
@@ -0,0 +1,27 @@
+name: Windows GitHub CI
+
+on: [pull_request, push]
+
+jobs:
+  minimal:
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: ilammy/msvc-dev-cmd@v1
+    - name: config
+      run: |
+        mkdir _build
+        cd _build
+        perl ..\Configure no-makedepend no-bulk no-deprecated no-fips no-asm -DOPENSSL_SMALL_FOOTPRINT VC-WIN64A
+        perl configdata.pm --dump
+        cd ..
+    - name: build
+      run: |
+        cd _build
+        nmake
+        cd ..
+    - name: test
+      run: |
+        cd _build
+        nmake test VERBOSE_FAILURE=yes TESTS=-test_fuzz
+        cd ..
```

Comment:
Something I would suggest is to have a separate preparation task to create the build directory, then use `working-directory:` with the remaining tasks...  so, something like this:

``` yaml
    steps:
    - uses: actions/checkout@v2
    - uses: ilammy/msvc-dev-cmd@v1
    - name: prepare the build directory
      run: mkdir _build
    - name: config
      working-directory: _build
      run: |
        perl ..\Configure no-makedepend no-bulk no-deprecated no-fips no-asm -DOPENSSL_SMALL_FOOTPRINT VC-WIN64A
        perl configdata.pm --dump
    - name: build
      working-directory: _build
      run: nmake
    - name: test
      working-directory: _build
      run: nmake test VERBOSE_FAILURE=yes TESTS=-test_fuzz
 ```

Reply:
yep - good idea..